code_generator_funcadl_uproot/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "code-generator-funcadl-uproot"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+packages = [{include = "code_generator_funcadl_uproot"}]
+
+[tool.poetry.dependencies]
+python = "~3.7"
+servicex-code-gen-lib = "1.1.0a1"
```

Comment:
Let's cut a real 1.1.0 release of this library

Reply:
<img width="1046" alt="Screenshot 2022-10-31 at 1 52 24 PM" src="https://user-images.githubusercontent.com/61829156/199087088-4ea0c788-f937-447f-876e-a53b8445a795.png">

Reply:
<img width="1440" alt="Screenshot 2022-10-31 at 1 58 14 PM" src="https://user-images.githubusercontent.com/61829156/199088107-b78e3a1b-0533-447f-b188-0198b88b934a.png">

The new release can be found on pypi